server/server.js

```diff
@@ -10,14 +10,16 @@ const jsonBodyParser = bodyParser.json()
 const cookieParser = require('cookie-parser')
 const session = require('express-session')
 const Mustache = require('mustache')
-const helpers = require('brave-alert-lib').helpers
+const { ALERT_STATE, helpers } = require('brave-alert-lib')
 
 const BraveAlerterConfigurator = require('./BraveAlerterConfigurator.js')
 const db = require('./db/db.js')
 
 const FLIC_THRESHOLD_MILLIS = 210 * 1000
 const HEARTBEAT_THRESHOLD_MILLIS = 75 * 1000
 const PING_THRESHOLD_MILLIS = 320 * 1000
+const SESSION_RESET_TIMEOUT = 7200 * 1000
+const REMINDER_MESSAGE_THRESHOLD = 120 * 1000
```

Comment:
I wasn't sure if that was overkill or not, but it makes what it is more obvious to me too so I can definitely update that :)

Reply:
Oh right, I also just realised looking at this more that I'm not sure the name of the 2nd one is descriptive for what it is, I'm thinking maybe `SUBSEQUENT_URGENT_MESSAGE_THRESHOLD` based on the task name in clickup ("Also send subsequent urgent request messages based on time since last message")? Cause it's not really a reminder in my mind if it's a new urgent message that says how many presses have happened

Reply:
Very true. `SUBSEQUENT_URGENT_MESSAGE_THRESHOLD` works for me.